.gitignore

```
@@ -1,2 +1,2 @@
-
-.DS_Store
+node_modules
+.DS_Store
```

```suggestion
**/.DS_Store
```
If you want to include this in our `.gitignore`, let's make sure we also ignore .DS_Store files that could appear in subdirectories.

---

Or add `.DS_Store` to the `.gitignore` in our `packages/contracts` folder. It's already in the `.gitignore` in `packages/ui`